docs/disclosure-log.md

```diff
@@ -0,0 +1,21 @@
+# Disclosure Log for IBC Protocols
+
+This document is a record of all the bugs or issues we uncovered while specifying & formally verifying the IBC protocols.
+
+
+##### 1. ICS3 deadlock due to relayer algorithm
```

Comment:
As I understand, this is a mistake in the example relayer algorithm described in ICS 18, not a mistake in ICS 3 itself - am I understanding correctly? Or is there code in ICS 3 which needs to be fixed as well?

Reply:
Just needs fix in ICS-18

Reply:
Added a remark clarifying that only ICS018 requires modifications.

For completeness, I added a separate issue in cosmos/ics for tracking the required changes in ICS018: https://github.com/cosmos/ics/issues/436